plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -146,17 +146,48 @@ const generateNextReviewAssignments = async ({
   console.log('Next level reviewers', nextLevelReviewers)
   const reviewAssignments: ReviewAssignmentObject = {}
 
+  // Check if other reviewAssignment is already assigned to create new ones LOCKED
+  const existingReviewAssignments: ExistingReviewAssignment[] =
+    await db.getExistingReviewAssignments(applicationId, stageNumber, nextReviewLevel)
+
+  const isLocked = existingReviewAssignments.some(
```

Comment:
Maybe just change the variable name now to something like "reviewAlreadyAssigned" ? It's a bit confusing because you also have "isLocked" below which means something similar but not the same.
 "shouldBeLocked" might be better, but why not just call it exactly what it is?

Reply:
Changing to `reviewAlreadyAssigned`

Reply:
Great, that's perfect. :)